src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/InvControl.java

```diff
@@ -124,14 +125,14 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
 	}
 
 	public void adminCloseInventory(InventoryCloseEvent event) {
-		if (event.getPlayer() != null && this.adminsWithInv.contains(event.getPlayer().getUniqueId())) {
+		if (this.adminsWithInv.contains(event.getPlayer().getUniqueId())) {
```

Comment:
Going to leave this null check off, but I suspect it was there with a reason too, just nothing as specific coming to mind as with the others.